src/main/java/io/vertx/ext/jdbc/spi/JDBCColumnDescriptorProvider.java

```diff
@@ -33,20 +33,6 @@
 @FunctionalInterface
 public interface JDBCColumnDescriptorProvider {
 
-  /**
-   * Create provider by parameter statement
-   *
-   * @param statement the prepared statement
-   * @return a new {@code JDBCTypeProvider} instance
-   * @see java.sql.PreparedStatement
-   */
-  static JDBCColumnDescriptorProvider fromParameter(PreparedStatement statement) {
```

Comment:
@pmlopes  You removed it completely, thanks! :)

Reply:
@ruslansennov sorry if you miss it. I was under the impression we can achieve the same with:

```
ParameterMetaData md = new CachedParameterMetaData(statement);
JDBCColumnDescriptorProvider provider = JDBCColumnDescriptorProvider.fromParameterMetaData(md);
```

I should add it back as this was an SPI not an impl

Reply:
It's reverted!

Reply:
@pmlopes  no! My personal opinion: it was the right decision :) But this method should be used very carefully.